libs/ui-lib/tsconfig.eslint.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Why do we need this? What was your intent?

Reply:
We `do not` want the test files in the build. We `do` want to lint the test files.

The linter needs to have all files matching for linting to be included in the tsconfig selection. One way how to tackle it is creating a new tsconfig for linter while reusing the "main" tsconfig and just easing the `exclude` section.

Reply:
https://github.com/openshift-assisted/assisted-installer-ui/pull/2226/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR52

https://github.com/openshift-assisted/assisted-installer-ui/pull/2226/files#diff-f98ebae61e31cb1a9fff4b3bb9f3475a7a7b44b5a1e8332eca456a9e0e85f833R23